src/main/java/org/opentripplanner/standalone/OTPMain.java

```diff
@@ -173,6 +176,18 @@ private static void startOTPServer(CommandLineParameters cli) {
     }
   }
 
+  /**
+   * Optionally, check if the config is valid and if not abort the startup process.
+   */
+  private static void detectUnusedConfigParams(CommandLineParameters cli, ConfigModel config) {
+    if (cli.configCheck && config.hasIUnknownProperties()) {
```

Comment:
I think the term used in the configuration module for a specific "property" is `parameter`, so this method should be renamed: `hasUnknownParameters`. Also, what is the `I` in `hasIUnkown`?

This comment apply to doc and other methods as well

Reply:
ba8e563a5f3e0572808d1ed41a4dcbea6dd7b23c

The stray I was just a typo... :flushed: .

Reply:
"property" -> parameter

Reply:
Sorry, that was a bit silly. I fixed it now: 5aeeffa2a953c9e80e4c66d2cec7ca83afe58114